### PR TITLE
ui bugz for landing page

### PR DESCRIPTION
**Pull request by ZeeJab**
This corrects all the ui bugs in the dashboard as per section 1) of #227 and these redpens: 
https://redpen.io/kn6a8fa0bfff0b912b
https://redpen.io/gvbe76faf89894c38d
https://redpen.io/qm874694931577560e

section 2) and 3) will be done in another pull request

### CODE REVIEW

File: clientapp/styles/button.scss [hunk 1]
```diff
@@ -6,15 +6,41 @@
...
+.button-square {
+  padding: 6px 13px;
+  background-color: white;
+  &:hover { background-color: $orange; }
+}
+
+.button-orange {
+  background: url(/static/button-orange.svg) no-repeat;
+  @extend .button-base
+}
```
Comment (stenington) [inline clientapp/styles/button.scss:36]: My complaint about including colors in class names is that if the color changes from orange to teal, we either have to do a big find/replace on "button-orange" to make it "button-teal", or we have to live with "button-orange" actually being teal which can be misleading.

I'd prefer to name it with a word that describes why it's orange. Is it a callout? A primary action? Then `.button-callout` or `.button-primary` might be better in the long run. That said I don't know if `.button-alert` was right for orange buttons either. I think of alert as reserved for dismissing errors or confirming permanent deletion or something.

File: clientapp/styles/dashboard-page.scss [hunk 2]
```diff
@@ -1,3 +1,16 @@
+.nav-container {
+  margin-top: 5px;
+  .fa-bars { color: black; }
+}
+
+.avatar-image { margin-right: 10px; }
+
+.user-email {
+  color: white;
+  font-family: "Open Sans";
```
Comment (stenington) [inline clientapp/styles/dashboard-page.scss]: Is it worth adding a few fallback fonts? I'm never sure what the best set of fonts to use is, but I think some other Mozilla properties may have worked out a string of fallbacks we could copy.

File: clientapp/styles/dashboard-page.scss [hunk 3]
```diff
@@ -9,5 +22,8 @@
     background: white;
     padding-top: 10px;
   }
+}
 
+@media screen and (max-width: 1024px) {
+  .nav-container { margin-left: 20px; }
 }
```
Comment (stenington) [inline clientapp/styles/dashboard-page.scss:29]: We could potentially set this .nav-container rule as the default, and then use `@media #{$large-up} {  }` to wrap a rule setting it to whatever it should be on desktop screens instead. The benefit would be that if we ever tweaked Foundation's responsive breakpoint widths this rule would continue to work without modification, although I also think that kind of tweaking is unlikely so it's not a big deal. I'm fine either way.

**Comment by stenington** [comment]
@ZeeJab Let me know what you think about the comments above. If you're happy without any additional changes then so am I.

File: clientapp/styles/button.scss (hunk 1 above)
Comment (ZeeJab) [inline clientapp/styles/button.scss:36]: You're right, you're right. ... I'll go with `.button-callout` for now. I think it's fitting.

File: clientapp/styles/dashboard-page.scss (hunk 2 above)
Comment (ZeeJab) [inline clientapp/styles/dashboard-page.scss]: yup, no probz

File: clientapp/styles/dashboard-page.scss (hunk 3 above)
Comment (ZeeJab) [inline clientapp/styles/dashboard-page.scss:29]: hmm, i think either way works. Let's leave it for now.

**Comment by ZeeJab** [comment]
@stenington I think this is good to go in its current state

**Pull request merged**